File: backend/src/main/java/org/globe42/dao/TaskCategoryDao.java
package org.globe42.dao;

import org.globe42.domain.TaskCategory;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * DAO for {@link org.globe42.domain.TaskCategory}
 * @author JB Nizet
 */
public interface TaskCategoryDao extends JpaRepository<TaskCategory, Long> {
}

File: backend/src/main/kotlin/org/globe42/dao/TaskCategoryDao.kt
package org.globe42.dao

import org.globe42.domain.TaskCategory
import org.springframework.data.jpa.repository.JpaRepository

/**
 * DAO for [org.globe42.domain.TaskCategory]
 * @author JB Nizet
 */
interface TaskCategoryDao : JpaRepository<TaskCategory, Long>
